advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -595,6 +603,165 @@
    "source": [
     "predictor.delete_endpoint()"
    ]
+  },
```

Comment:
Would it be possible to add a section showing how to use AutoGluon's built-in permutation feature importance scores for interpretability as well?

https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-indepth.html#interpretability-feature-importance

I would structure such a section like this:

## Interpretability (permutation feature importance)

To better understand our trained predictor, we can estimate how much each feature contributes to the predictor's accuracy via feature importance scores:

```
predictor.feature_importance(test_data)
```
For more details/options, see the AutoGluon feature-importance tutorial: https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-indepth.html#interpretability-feature-importance